### PR TITLE
Release/v9.1.2

### PR DESCRIPTION
**Pull request by jemten**
### This PR adds | fixes:

- Merges the release branch back into develop

### How to test:

- Automatic and continuous test suite

### Expected outcome:
- [ ] Installation, unit and integration test suite pass

### Review:
- [ ] Code review
- [ ] Tests pass

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes. If applicable record manual test results in PR header
- [ ] **MINOR** - when you add functionality in a backwards compatible manner. If applicable record manual test results in PR header
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: lib/MIP/Recipes/Analysis/Markduplicates.pm [hunk 1]
```diff
@@ -367,23 +348,23 @@ sub analysis_markduplicates {
...
-                case_id              => $case_id,
-                dependency_method    => q{sample_to_sample},
-                job_id_chain         => $job_id_chain,
-                job_id_href          => $job_id_href,
-                job_reservation_name => $active_parameter_href->{job_reservation_name},
-                log                  => $log,
+                base_command                      => $profile_base_command,
+                case_id                           => $case_id,
+                dependency_method                 => q{sample_to_sample},
+                job_id_chain                      => $job_id_chain,
```
Comment (henrikstranneheim) [inline lib/MIP/Recipes/Analysis/Markduplicates.pm:364]: This needs to be updated to use recipe{}.

Comment (jemten) [reply lib/MIP/Recipes/Analysis/Markduplicates.pm:364]: Yeah. I expect running into all sorts of weird things. Develop has changed a lot

**Review by henrikstranneheim — APPROVED** [review]
Well done!

**Review by ramprasadn — APPROVED** [review]
👍🏻

**Pull request merged**